source/common/router/config_impl.h

```diff
@@ -140,6 +140,10 @@ class VirtualHostImpl : public VirtualHost {
   const RateLimitPolicy& rateLimitPolicy() const override { return rate_limit_policy_; }
   const Config& routeConfig() const override;
   const RouteSpecificFilterConfig* perFilterConfig(const std::string&) const override;
+  Upstream::RetryPrioritySharedPtr retryPriority() override { return nullptr; } // TODO: implement
```

Comment:
Please add your github username as part of the TODO (e.g. `// TODO(snowp): implement`)